sidestep_engine/gui/gpu_monitor.py

```
@@ -61,15 +86,18 @@ def _snapshot_one(gpu_index: int) -> Dict[str, Any]:
-        import pynvml
-        count = pynvml.nvmlDeviceGetCount()
-        return [_snapshot_one(i) for i in range(count)]
-    except Exception as exc:
-        logger.debug("GPU enumeration failed: %s", exc)
-        return []
+    if _ensure_nvml():
+        try:
+            import pynvml
+            count = pynvml.nvmlDeviceGetCount()
+            return [_snapshot_one(i) for i in range(count)]
+        except Exception as exc:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)